### PR TITLE
feat(thirdparty): update payloads for `authorizations` and `verifications` resources

### PR DESCRIPTION
**Pull request by lewisdaly**
Part of mojaloop/project#2372

- include `FIDOPublicKeyCredentialAssertion` payload

Note: these are updated along with the Data Model document in Draft Pr Status here: https://github.com/mojaloop/mojaloop-specification/pull/90

I'll update the sequences in the pisp repo once we are happy and merge this guy in.

### CODE REVIEW

File: thirdparty/openapi3/components/schemas/ThirdpartyRequestsVerificationsPostRequest.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-title: ThirdpartyRequestsVerificationsPostRequest
+title: ThirdpartyRequestsTransactionAuthenticationValue
```
Comment (kleyow) [inline thirdparty/openapi3/components/schemas/ThirdpartyRequestsVerificationsPostRequest.yaml:1]: nit: title?

File: tests/dto/thirdparty.test.ts [hunk 2]
```diff
@@ -592,13 +592,18 @@ describe('thirdparty', () => {
...
       verificationRequestId: correlationId,
       challenge: 'some challenge base64 encoded',
-      value: {
-        authenticationInfo: {
-          authentication: 'U2F',
-          authenticationValue: 'some signed challenge base64 encoded'
-        }
-      },
-      consentId: correlationId
+      consentId: '8d34f91d-d078-4077-8263-2c0498dhbjr',
```
Comment (kleyow) [inline tests/dto/thirdparty.test.ts:595]: aya...meaning we need to add fetching and creating dfsp backend calls for this id.

File: thirdparty/openapi3/paths/thirdpartyRequests_verifications.yaml [hunk 3]
```diff
@@ -32,14 +32,16 @@ post:
         example:
           verificationRequestId: '44444444-0000-0000-0000-000000000000'
```
Comment (kleyow) [inline thirdparty/openapi3/paths/thirdpartyRequests_verifications.yaml:33]: nit: this has been changed to `consentId`, yes?

File: thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml [hunk 4]
```diff
@@ -32,23 +32,29 @@ post:
           authorizationRequestId: 33333333-0000-0000-0000-000000000000
           transactionRequestId: '00000000-0000-0000-0000-000000000000'
           challenge: '<base64 encoded binary - the encoded challenge>'
-          quote:
```
Comment (kleyow) [inline thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml:35]: remind me why `quote`'s fields are being brought up to root level?

File: thirdparty/openapi3/paths/thirdpartyRequests_verifications.yaml (hunk 3 above)
Comment (kleyow) [reply thirdparty/openapi3/paths/thirdpartyRequests_verifications.yaml:33]: ignore

File: thirdparty/openapi3/components/schemas/ThirdpartyRequestsTransactionsIDAuthorizationsPostRequest.yaml [hunk 5]
```diff
@@ -1,3 +1,4 @@
+# TODO: should this be removed?
```
Comment (kleyow) [inline thirdparty/openapi3/components/schemas/ThirdpartyRequestsTransactionsIDAuthorizationsPostRequest.yaml:1]: Yes. We can prune these resources/components at a later date.

File: thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml (hunk 4 above)
Comment (lewisdaly) [reply thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml:35]: The CCB didn't like us disclosing private details about the quote that aren't relevant to the PISP. We also don't need things such as condition, ilppacket etc.

File: thirdparty/openapi3/components/schemas/ThirdpartyRequestsTransactionsIDAuthorizationsPostRequest.yaml (hunk 5 above)
Comment (lewisdaly) [reply thirdparty/openapi3/components/schemas/ThirdpartyRequestsTransactionsIDAuthorizationsPostRequest.yaml:1]: Cool.

File: thirdparty/openapi3/components/schemas/ThirdpartyRequestsVerificationsPostRequest.yaml (hunk 1 above)
Comment (lewisdaly) [reply thirdparty/openapi3/components/schemas/ThirdpartyRequestsVerificationsPostRequest.yaml:1]: Oops!

**Review by kleyow — APPROVED** [review]

**Pull request merged**